models/categoryDataModel.js

```diff
@@ -58,3 +58,27 @@ exports.fetchReviewById = (review_id, sort_by) => {
       }
     });
 };
+
+exports.fetchCommentsByReviewId = (review_id, sort_by) => {
+  if (isNaN(review_id)) {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
```

Comment:
I think if you send this to the DB without checking here the db will reject and you can handle this by adding a function to  your error handling middleware chain